package.json

```diff
@@ -765,6 +765,11 @@
 					"description": "Command to execute or full path to folder and executable to start the Mongo shell, needed by some Mongo scrapbook commands. If empty, will search in the system path for 'mongo'.",
 					"default": null
 				},
+				"mongo.shell.timeoutInMilliseconds": {
+					"type": "number",
+					"description": "The duration allowed (in milliseconds) for the mongo shell to execute a command. Default value is 5 seconds",
```

Comment:
Mongo

Reply:
I like "inMilliseconds" for variable names, but not so sure users will like it for options. I'd go with simple mongo.shell.timeout

Reply:
Mongo in 'Mongo shell'?

Reply:
If you're going to get rid of "inMilliseconds" I would make the timeout based on seconds. I feel like that's much more common for these types of settings

Reply:
Current settings are mixed:
![image](https://user-images.githubusercontent.com/6913354/47517464-29c23c00-d83d-11e8-9667-251d4ad00246.png)

But I'm good with seconds.